front/public/locales/de/operationalStudies/drivertrainschedule.json

```
@@ -0,0 +1,27 @@
+{
+  "actualspeed": "Tatsächlich",
+  "actualspeed-short": "Tat.",
+  "averagespeed": "Durchschnitt",
+  "averagespeed-short": "Dur.",
+  "composition": "Zusammensetzung",
+  "dcmdetails": "Zugverlauf n°",
+  "destination": "Ziel",
+  "energyconsumed-basic": "Kürzeste Laufzeit",
+  "energyconsumed-eco": "Energieschonendste Laufzeit",
```

```suggestion
  "energyconsumed-basic": "Kürzeste Fahrzeit",
  "energyconsumed-eco": "Energieeffizienteste Fahrzeit",
```

(not sure for "Energieschonendste" -> "Energieeffizienteste"

---

"...schonendste" is slightly less formal; "...effizienteste" will be fully understandable too.